server/sonar-web/src/main/js/apps/custom-measures/custom-measures.js

```diff
@@ -60,7 +60,7 @@ export default Backbone.Collection.extend({
   },
 
   getTakenMetrics () {
-    const metrics = this.map(function (model) {
+    const metrics = this.map(model => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)